apps/i18n/common/en_us.json

```diff
@@ -523,6 +523,12 @@
   "findLocalClassButton": "Find a class",
   "finish": "Finish",
   "forTeachersOnly": "For Teachers Only",
+  "gdprDialogHeader": "Do you agree to using a web site based in the United States?",
+  "gdprDialogDetails": "Data from your use of this site may be sent to and stored or processed in the United States.",
+  "gdprDialogLinkIntro": "For details see our",
```

Comment:
I don't think recombining a single sentence will localise well.

Reply:
Also fixed in my PR to make this a single sentence and make the whole sentence a link.